tests/sdk/test_client.py

```diff
@@ -199,3 +199,49 @@ def test_client_configuration_env_var_overrides_config_file(mocker, monkeypatch,
 def test_login_remote_without_url(tmpdir):
     with pytest.raises(exceptions.SDKException):
         Client(backend_type="remote")
+
+
+def test_client_configuration_configuration_file_path_from_env_var(mocker, monkeypatch, config_file):
+    """
+    The configuration file path can be set through an env var
+    """
+    mocker.patch("substra.sdk.Client.login", side_effect=stub_login)
+    monkeypatch.setenv("SUBSTRA_CLIENTS_CONFIGURATION_FILE_PATH", config_file)
```

Comment:
I would suggest storing the value of this string into a global variable, but I'm afraid we'd argue on how to name it ^^